server/app/Listeners/CommentCreatedNotification.php

```diff
@@ -30,8 +30,8 @@ public function handle(CommentCreated $event)
         // if the comment is in reply to something, signal a notification to
         // the parent user
         if (!empty($event->parent)) {
-            if ($event->parent instanceof \App\Models\Annotation) {
-                $parentType = 'annotation';
+            if ($event->parent->isNote()) {
```

Comment:
Getting `Call to undefined method Illuminate\Database\Query\Builder::isNote()` on this.